gradle-plugin/doc/src/main/asciidoc/inc/tasks/build/_jkube-build.adoc

```diff
@@ -34,6 +35,113 @@ These are the different options supported by `buildStrategy`:
 |===
 
 {plugin} by default tries to build up an opinionated <<config-image, Image Configuration>> by inspecting `build.gradle`. You can also provide your own <<dockerfile-scenario, Dockerfile>> or provide Custom ImageConfiguration via <<groovy-scenario-image, Groovy DSL configuration>>.
+endif::[]
```

Comment:
I think you added this by accident

Reply:
I wanted to have "Kubernetes Build" section visible only in case of `task-prefix == k8s` and "OpenShift Build" when `task-prefix == oc`. Hence I added an extra if-else here around Kubernetes Build section.

I can remove it if you think it's not necessary.

Reply:
I'm referring specificaly to line 38. You added an extra `endif` (I think).

Reply:
Not sure why it's not appearing property on Github UI. I think `endif` on line 38 is closing `ifeval::["{task-prefix}" == "k8s"` on line 6.

Reply:
OK, I missed that one, was searching for an oc instead